src/Objs/SearchResults/SearchResultItem.js

```diff
@@ -12,6 +12,7 @@ const PreviewImage = Scrivito.connect(({ item }) => {
     item.get("navigationBackgroundImage") ||
     item.get("titleImage") ||
     item.get("image");
+  if (image instanceof Scrivito.Binary) return null;  
   if (!image || isVideoObj(image)) return null;
```

Comment:
Please check, that image is (not) an Obj. Otherwise every other attribute time other than `binary` would also fail this component. And to make the code flow clearer I would split each guard to its own line:

```suggestion
  if (!image) return null;
  if (!(image instanceof Scrivito.Obj)) return null;
  if (isVideoObj(image)) return null;
```